plotly/figure_factory/_dendrogram.py

```diff
@@ -144,7 +145,7 @@ def __init__(self, X, orientation='bottom', labels=None, colorscale=None,
         self.zero_vals.sort()
 
         self.layout = self.set_figure_layout(width, height)
-        self.data = dd_traces
+        self.data = graph_objs.Data(dd_traces)
```

Comment:
`graph_objs.Data` is deprecated in version 3. Just stick with `dd_traces` as a `list` or `tuple`.